base_phone_rate/models/phone_rate.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (C) 2019 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+from odoo.addons import decimal_precision as dp
+
+
+class PhoneRate(models.Model):
+    _name = "phone.rate"
```

Comment:
It is good practice for the first word to refer to the App or domain the Model belongs to. I suggest:
```suggestion
    _name = "isp.phone.rate"
```